packages/smooth_app/lib/helpers/permission_helper.dart

```diff
@@ -5,11 +5,13 @@ import 'package:permission_handler/permission_handler.dart';
 class PermissionListener extends ValueNotifier<DevicePermission> {
   PermissionListener({
     required this.permission,
-  }) : super(DevicePermission._initial(permission)) {
+  })  : _status = _DevicePermissionStatus.initial,
+        super(DevicePermission._initial(permission)) {
     checkPermission();
   }
 
   final Permission permission;
+  _DevicePermissionStatus _status;
```

Comment:
```suggestion
  _DevicePermissionStatus _status = _DevicePermissionStatus.initial;
```
```suggestion
  _DevicePermissionStatus _status;
```

Reply:
It's already done line 8.
What would that change?

Reply:
For the computer, nothing.
For the person who reads your code, that would mean that each time an instance of the object is created, it starts from scratch with `initial`, regardless of the constructor. And that has nothing to do with that specific constructor that would handle a specific case.